modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/PagerDotIndicator.kt

```diff
@@ -0,0 +1,91 @@
+package au.com.shiftyjelly.pocketcasts.compose
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.pager.PagerState
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.IntOffset
+import androidx.compose.ui.unit.dp
+import kotlin.math.absoluteValue
+import kotlin.math.roundToInt
+import kotlin.math.sign
+
+private val dotWidth = 8.dp
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun PagerDotIndicator(
```

Comment:
Is it possible to combine `PagerDotIndicator` and existing `PageIndicator` components?

Reply:
Works for me.